### PR TITLE
Development

### PR DESCRIPTION
**Pull request by JamesThreadgill97**

### CODE REVIEW

**Comment by JamesThreadgill97** [comment]
Added files to json file, just wanting to implement on master branch

File: back-end/historicalFigures.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[{"name":"William Shakespeare", "hint1": "To be or not to be, that is the question", "hint2":"He is famous for writing plays", "funFact": "Even though he wrote at least 37 plays, and collaborated on many more, William Shakespeare couldn't actually spell!"},
+{"name":"Charles Darwin", "hint1":"He was a scientist from the 1800s", "hint2":"There are disputes between him and other scientists as to who discovered their topics firs", "funFact":"Before working on evolution and natural selection, Charles Darwin was studing at medical school, but had to drop out as he couldn't stand the sight of blood!"},
```
Comment (EvansD13) [inline back-end/historicalFigures.json]: "...discovered their topics *first*" --typo

File: back-end/historicalFigures.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+{"name":"Guy Fawkes", "hint1":"Remember, remember, the fith of November!", "hint2":"Gunpowder plot", "funFact":"In 1592, aged 21, Fawkes sold the estate he had inherited and set sail for Europe to join the Catholic Spanish army, to help wrench the Netherlands back from the Protestant Dutch forces. Rising through the ranks of the Spanish military, he was promoted to captain and developed great knowledge of gunpowder, which would come in handy later. He also adopted the Italian name, 'Guido'."},
+{"name":"King Sweyn Forkbeard", "hint1":"The son of a Danish king, and became King of England for five week until his death", "hint2":"Born in 963 AD and died in 1014", "funFact":"The son of a Danish king, Sweyn 'Forkbeard' began conquering territory in England in 1003, effectively devastating much of southern and midland England. The English nobility became so disillusioned with their existing king, Ethelred 'The Unready', that they acknowledged Sweyn as king in 1013."},
```
Comment (EvansD13) [inline back-end/historicalFigures.json]: "..five *weeks*

**Review by EvansD13 — APPROVED** [review]
Spotted a couple of typos, but looks good!

**Pull request closed without merging**